crates/bevy_dev_tools/src/fps_overlay.rs

```diff
@@ -128,9 +131,10 @@ fn customize_text(
     mut writer: UiTextWriter,
 ) {
     for entity in &query {
-        writer.for_each_style(entity, |mut style| {
-            *style = overlay_config.text_config.clone();
+        writer.for_each_font(entity, |mut font| {
+            *font = overlay_config.text_config.clone();
         });
+        writer.for_each_color(entity, |mut color| color.0 = overlay_config.text_color);
```

Comment:
Can we keep `for_each_style` and have it pass both the font and color as params?

Reply:
We could but I'm wondering if all these helper functions are even needed though. Users can just use `for_each` and ignore the fields that they don't need? We could return a struct with named fields instread of an enum tuple to make it more ergonomic.

Reply:
Yeah, I believe that we should leave this helpers to a follow-up PR so we can discuss better what to do with them